Formula/ruby@2.7.rb

```diff
@@ -28,7 +28,6 @@ class RubyAT27 < Formula
 
   depends_on "pkg-config" => :build
   depends_on "libyaml"
-  depends_on "openssl@1.1"
```

Comment:
How is the dependency on OpenSSL 1.1 expressed in Ruby 2.7 if this line is removed? Is it possible for you to add a dependency on our Formula instead of the core Homebrew formula?

Reply:
Not sure if I can do add the dependency over here but I have added a block in puppet-runtime with which the runtime build was successful on local. Let me know if that looks okay.
`if platform.is_macos? && platform.is_cross_compiled?
  pkg.build_requires "skyamgarp/puppet/openssl@1.1"
else
  pkg.build_requires "openssl-#{settings[:openssl_version]}"
end`